workshop/Lab0/README.md

```diff
@@ -52,13 +57,14 @@ $echo $PATH
 
 3. Convert the binary file to an executable: `chmod +x /usr/local/bin/kubectl`
 
-# Download the Workshop Source Code
-Repo `guestbook` has the application that we'll be deploying. 
-While we're not going to build it we will use the deployment configuration files from that repo. 
-Guestbook application has two versions v1 and v2 which we will use to demonstrate some rollout 
+## Download the workshop source code
+
+The [`guestbook`](https://github.com/IBM/guestbook) repo contains the application that we'll be deploying. 
+While we're not going to build it as part of these labs, we will use the deployment configuration files from that repo. 
+The guestbook application has two versions, v1 and v2, which we will use to demonstrate some rollout 
 functionality later. All the configuration files we use are under the directory guestbook/v1.
 
-Repo `kube101` contains the step by step instructions to run the workshop.
+The [`kube101`](https://github.com/IBM/kube101/) contains the step-by-step instructions to run the workshop.
```

Comment:
Didn't notice this before, but this sentence feels out of place doesn't it? We're already in the kube101 repo so why are we mentioning it as part of downloading "guestbook" ? If we want to keep this sentence, shouldn't it go near the top of this README, or in the repo's main README, to explain what this doc is for?

Reply:
Could we say, "Proceed to the following labs within this repo for the next steps" ?
Otherwise, we can remove.

Reply:
@iamjinakim  Most of workshop is markdown but there is at least few artifacts in it ex: lab3 has healthcheck.yaml, etc. We need to specify that they still need to `git clone` locally. This section makes sense to me even if someone thinks they can read markdowns online and run the workshop.
I will be ok if you reword to convey this.